docs/developer-docs/latest/plugins/upload.md

```diff
@@ -540,3 +541,31 @@ In our second example, you can upload and attach multiple pictures to the restau
   // ...
 }
 ```
+
+### Update file info
```

Comment:
@torchsmith What do you think about changing info to "information" here and in the table? Otherwise, it could be `fileInfo` since that is what is in the code example. After this it looks good and could be deployed as soon as tomorrow.

Reply:
Let's have it setup as fileInfo to be more consistent with the code.